src/gc/sample/gcenv.ee.cpp

```diff
@@ -129,7 +129,8 @@ void ThreadStore::AttachCurrentThread()
     g_pThreadList = pThread;
 }
 
-void GCToEEInterface::SuspendEE(GCToEEInterface::SUSPEND_REASON reason)
+#ifndef FEATURE_STANDALONE_GC
+void GCToEEInterface::SuspendEE(SUSPEND_REASON reason)
```

Comment:
You're right, this fixed a break in the sample when the GCToEEInterface class that I removed was defined. Will fix.